### ISSUE TITLE
Request: Timestamp copy button

### ISSUE THREAD

**Issue by hi-im-karma** [arichornlover/uYouEnhanced#401]
Would be very useful if we had this button from Revanced Extended:

![Screenshot_20240228-092949694 (1).jpg](https://github.com/arichornlover/uYouEnhanced/assets/73917917/d9c98130-72e3-4456-8864-29264b8a7cc1)

**Comment by KarmaBeast** [comment]
Is that something you could implement? @arichornlover

**Comment by KarmaBeast** [comment]
I used this feature so often with my Android. Every day sharing videos to my friends and copying the timestamp of the exact point I want them to see in the video. So when they open the link the video starts exactly there.

It's so damn useful to have.

**Comment by KarmaBeast** [comment]
Ok understood. Thanks for keeping me posted. Having this feature would be awesome!

**Comment by KarmaBeast** [comment]
@arichornlover could you please consider prioritizing this request?

None of my iOS shortcuts are working anymore when I switched over to non-official YouTube (enhanced)

None of these work:
![image](https://github.com/arichornlover/uYouEnhanced/assets/73917917/77f1770d-ead7-4b31-a775-f1ffe0e22b28)

**Comment by KarmaBeast** [comment]
I already spent so much time trying the iOS shortcuts with enhanced, but they simply don't work anymore.

It appears they only work with AppStore YouTube...

**Comment by bhackel** [comment]
@arichornlover how is progress on this tweak going?

**Comment by KarmaBeast** [comment]
Any good news on this? 🙏🏻

**Comment by aaronliu0130** [comment]
@KarmaBeast At least https://www.icloud.com/shortcuts/7ecdbce4df394880954cda3b6de5702b works with me

**Comment by aaronliu0130** [comment]
Yeah, but in the meantime you should be able to summon the shortcut from the iOS share dialog

**Comment by aaronliu0130** [comment]
Uhh, which one is it? The speedy clock seems like it adjusts the playback speed.

**Comment by aaronliu0130** [comment]
The copy icon seems to be the copy icon (which hopefully one can find somewhere in YouTube like the share button or in FontAwesome if they're desperate) combined with a clock in the bottom right.

**Comment by aaronliu0130** [comment]
Stretch goal here: tweak to skip straight to the native one? :p 

Anyways just the button form would be nice!

**Comment by aaronliu0130** [comment]
I usually associate the menu icon with menus or right click. How about https://fontawesome.com/icons/copy?f=classic&s=regular + the clock?

**Comment by aaronliu0130** [comment]
> I couldn’t find the icon that ReVanced used in the screenshot above

**Comment by aaronliu0130** [comment]
Like I said, it doesn't tell "copy" to me.

**Comment by aaronliu0130** [comment]
You clearly weren't here before the release of 3.0 :) There were no updates for 3 entire years until 2023. Seems to me like they made the bad decisions to not open source or actually make downloading as fast as YoutubeReborn and then just ran out of passion.

**Comment by aaronliu0130** [comment]
Veering into off-topic areas here, but...

>3.0.3
>
>* Fixed startup crash.
>* Fixed minor UI issues.
>* Added "Hide Next/Previous Buttons".
>* Added "Disable Double Tap To Seek".
>* Added "Disable Startup To Shorts".
>* Resolved issue where downloaded video/audio files were not being deleted properly.

**Comment by bhackel** [comment]
@arichornlover What is the current issue? I have worked with timestamp stuff before in a few tweaks, maybe I can take a look

**Comment by bhackel** [comment]
@arichornlover It looks like it works on the top but crashes when on the bottom. This is the error: `exceptionReason" : {"arguments":["%s","playerViewController","0x10fd44ea0"],"format_string":"-[%s %s]: unrecognized selector sent to instance %p","name":"NSInvalidArgumentException","type":"objc-exception","composed_message":"-[%s playerViewController]: unrecognized selector sent to instance 0x10fd44ea0","class":"NSException"},`

**Comment by aaronliu0130** [comment]
It'll make *every* timestamped link invalid, so I doubt that we should worry about this.
